spring-boot-admin-server-ui/src/main/frontend/views/journal/index.vue

```diff
@@ -188,7 +188,11 @@ export default {
   computed: {
     instanceNames() {
       return this.events
-        .filter((event) => event.type === InstanceEvent.REGISTERED)
+        .filter(
+          (event) =>
+            event.type === InstanceEvent.REGISTERED ||
+            event.type === InstanceEvent.REGISTRATION_UPDATED,
+        )
         .reduce((names, event) => {
```

Comment:
```suggestion
        .reduceRight((names, event) => {
```

You should run through the events in reverse order (from newest to oldest) so that the most recent events set the names and older events no longer overwrite them.